adsmsg/metrics_record.py

```diff
@@ -23,7 +23,15 @@ def __init__(self, *args, **kwargs):
                 
 
 
-class MetricsRecordList(Msg):
+class MetricsRecordList():
     
-    def __init__(self, *args, **kwargs):        
-        super(MetricsRecordList, self).__init__(metrics_pb2.MetricsRecordList(), args, kwargs)
+    def __init__(self, metrics_records):
+        instance = metrics_pb2.MetricsRecordList()
+        self.__dict__['_data'] = instance  
+        for current in metrics_records:
+            tmp = MetricsRecord(**current)
+            current_protobuf = tmp.__dict__['_data']
+            instance.metrics_records.extend([current_protobuf])
```

Comment:
Indeed controversial PR

This line is doing the same as the next line (36); in the end it is wrapping itself - this can be rethought